browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -28,7 +28,10 @@ class MarkdownNoteDetail extends React.Component {
         content: ''
       }, props.note),
       isLockButtonShown: false,
-      isLocked: false
+      isLocked: false,
+      fullScreen: false,
+      widthOfNoteDetail: 0,
+      widthOfMainBody: 0
```

Comment:
Alright, let's move on to the architectural review!
First of all, these states should not be placed the component because they're out of the role of it.
I think `browser/main/Main.js` should manage them rather than `MarkdownNoteDetail` because those states are overall states for Boostnote, and you'll have to use `eventEmitter` to call `handleFullScreenButton()` from `MarkdownNoteDetail`.
I believe this PR (https://github.com/BoostIO/Boostnote/pull/283/files) is helpful for you!